src/ui/molecules/navigation/navigation.chain.dropdown.js

```diff
@@ -25,11 +32,30 @@ const chains = [
 ];
 
 export default function NavigationChainDropdown({ chainId, setChainId }) {
+  const { width } = useWindowDimensions();
+
+  const chainToIcon = (chain) => {
+    switch (chain.toLowerCase()) {
+      case "mainnet":
+        return <ETH className="h-5 w-5" />;
+      case "arbitrum":
+        return <ARB className="h-5 w-5" />;
+      case "polygon":
+        return <MATIC className="h-5 w-5" />;
+      case "avalanche":
+        return <AVAX className="h-5 w-5" />;
+    }
+  };
```

Comment:
We need to add a default case here for when the network is unsupported. Just a question mark icon will do it.

Reply:
Added default question mark